lib/matplotlib/image.py

```diff
@@ -514,8 +514,7 @@ def contains(self, mouseevent):
             ymin, ymax = ymax, ymin
 
         if x is not None and y is not None:
-            inside = ((x >= xmin) and (x <= xmax) and
-                      (y >= ymin) and (y <= ymax))
+            inside = xmin <= x <= xmax and ymin <= y <= ymax
```

Comment:
I agree with @qulogic.  Extra parentheses can greatly help readability, and in such cases, like this one, they do no harm at all.